doc/source/User_guide/pyaedt_extensions_doc/commandline.rst

```diff
@@ -4,8 +4,8 @@ Every extension can also launch the extension user interface from the terminal:
 
 .. code::
 
-   SET PYAEDT_SCRIPT_PORT=50051
-   SET PYAEDT_SCRIPT_VERSION=2025.2
+   SET PYAEDT_ELECTRONICS_PORT=50051
```

Comment:
This is open to discussion, but maybe the variable should be:

PYAEDT_DESKTOP_PORT
PYAEDT_DESKTOP_VERSION

OR

PYAEDT_SESSION_PORT
PYAEDT_SESSION_VERSION

What do you think @SMoraisAnsys @maxcapodi78 @eblanco-ansys ?